projects/plugins/jetpack/extensions/blocks/paywall/edit.js

```diff
@@ -1,15 +1,30 @@
-import { __ } from '@wordpress/i18n';
 import './editor.scss';
+import { useSelect } from '@wordpress/data';
+import { store as editorStore } from '@wordpress/editor';
+import { __ } from '@wordpress/i18n';
+import { accessOptions } from '../subscriptions/constants';
+import { GetAccessLevel } from '../subscriptions/utils';
```

Comment:
Given that `accessOptions` and `getAccessLevel` are used by multiple blocks (here and subscriptions), I wonder if those should be moved to someplace more general rather than importing from one block to another? Not sure where. Maybe /project/plugins/jetpack/extensions/shared?

Reply:
Yeah, I think we should move a bunch of shared code away from `subscriptions` to someplace other, and structure things under subscription form better, too.

We could start by having this under `shared/subscriptions` or similar?

Reply:
I was lazy to create a new file and moved all common methods to shared/memberships 😅